webservices/src/main/java/com/javaschool/webservices/controller/PackageController.java

```diff
@@ -0,0 +1,31 @@
+package com.javaschool.webservices.controller;
+
+import java.util.List;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.javaschool.webservices.service.PackageService;
+
+@RestController
+@RequestMapping("packageInformation")
```

Comment:
use a - instead of camelCase

Reply:
rest service name changed following REST conventions.